setup.py

```diff
@@ -53,16 +64,29 @@ def read(*rnames):
         + '\n' +
         'Download\n'
         '**********************\n'
-        ),
-
-    package_dir = {'':'src'},
-    packages = find_packages('src'),
-    include_package_data = True,
-    data_files = [('.', ['README.txt'])],
-    namespace_packages = ['zc', 'zc.recipe'],
-    install_requires = ['zc.buildout >=1.4', 'setuptools'],
-    extras_require = dict(test=['zope.testing']),
-    entry_points = {'zc.buildout':
-                    ['default = %s:Recipe' % name]},
-    zip_safe = True,
-    )
+    ),
+    package_dir={'':'src'},
+    packages=find_packages('src'),
+    include_package_data=True,
+    namespace_packages=['zc', 'zc.recipe'],
+    install_requires=[
+        'zc.buildout >= 1.4',
+        'setuptools'],
+    extras_require={
+        'test': [
+            # sadly zc.buildout doesn't have a test extra, so we
+            # need to duplicate its test dependencies, since we import its
+            # test package.
+            'zc.buildout[test]',
```

Comment:
You say that zc.buildout doesn't have a test extra, and then go ahead and use zc.buildout's test extra.  ... ?